test/cli/cli.test.js

```diff
@@ -705,6 +1035,61 @@ describe('CLI', () => {
       });
   });
 
+  it('--watch-files', (done) => {
+    const watchDirectory = path.resolve(__dirname, '../fixtures/static/static');
+
+    testBin(`--watch-files ${watchDirectory}`)
+      .then((output) => {
+        expect(output.exitCode).toEqual(0);
+        done();
+      })
+      .catch(done);
+  });
+
+  it('--watch-files-reset', (done) => {
+    const watchDirectory = path.resolve(__dirname, '../fixtures/static/static');
+
+    testBin(`--watch-files-reset --watch-files ${watchDirectory}`)
+      .then((output) => {
+        expect(output.exitCode).toEqual(0);
+        done();
+      })
+      .catch(done);
+  });
+
+  it('--watch-files-reset --watch-files-paths <value>', (done) => {
+    const watchDirectory = path.resolve(__dirname, '../fixtures/static/static');
+
+    testBin(`--watch-files-reset --watch-files-paths ${watchDirectory}`)
+      .then((output) => {
+        expect(output.exitCode).toEqual(0);
+        done();
+      })
+      .catch(done);
+  });
+
+  it('--watch-files-paths', (done) => {
+    const watchDirectory = path.resolve(__dirname, '../fixtures/static/static');
+
+    testBin(`--watch-files-paths ${watchDirectory}`)
+      .then((output) => {
+        expect(output.exitCode).toEqual(0);
+        done();
+      })
+      .catch(done);
+  });
+
+  it('--watch-files-paths-reset', (done) => {
+    const watchDirectory = path.resolve(__dirname, '../fixtures/static/static');
+
+    testBin(`--watch-files-paths-reset --watch-files-paths ${watchDirectory}`)
+      .then((output) => {
+        expect(output.exitCode).toEqual(0);
+        done();
+      })
+      .catch(done);
+  });
+
```

Comment:
Can we add test: using `webpack server --config ./path/webpack.config.js` with `--watch-files-paths-reset --watch-files-paths ${watchDirectory}` and do snapshot (you can set `stats: false`, we will improve it late), as you can see using `expect(output.exitCode).toEqual(0);` is bad testing, we need snapshot stdout and stderr, it allow use to see what we rewrite `watch-files-paths`

Reply:
We don't log which files are being watched on stderr/stdout

Reply:
Though we do log for `static`, added more snapshots 👍🏻